cogs/tickets.py

```diff
@@ -197,21 +245,19 @@ async def close(self, ctx):
             text="IsThicc Tickets"
         )
 
-        await self.db.execute('UPDATE tickets SET closing = false WHERE channel_id = ' + str(ctx.channel.id))
-        user = await self.db.execute("SELECT user_id FROM tickets WHERE channel_id = " + str(ctx.channel.id))
+        await asyncio.sleep(3)
 
-        msg    = await ctx.send(embed=closing)
-        user   = await self.bot.get_user(user[0])
-        closed = discord.utils.get(ctx.guild.categories, name="『 Archived Tickets 』")
-        time   = datetime.now().strftime("%A, %d %b %Y %l:%M %p")
+        await self.db.execute('UPDATE tickets SET open = false WHERE channel_id = ' + str(ctx.channel.id))
+        msg = await ctx.send(embed=closing)
+        closed = discord.utils.get(ctx.guild.categories, id=0) # TODO: Change this to the channel of the archived tickets category id
 
         await ctx.channel.edit(
             name=f"Archived-{user.display_name}",
-            topic=f"Ticket opened by {user.mention}. Closed by {ctx.author.mention} at {time}!",
+            topic=f"Ticket opened by {user.mention}. Closed by {ctx.author.mention}!",
             category=closed,
             reason="Ticket Closed."
         )
-        await asyncio.sleep(3)
+        await ctx.channel.set_permissions(ctx.author, send_messages=False, read_messages=False)
```

Comment:
Ticket creators should be able to read, and send messages in a closed ticket(in case they want to reopen it), I suggest just leaving the channel perms the same but moving it.